packages/l10n/src/create-l10n-injector/create-l10n-injector.spec.tsx

```diff
@@ -24,155 +23,74 @@ const candies: Candies = {
     3: 'gummibär',
   },
 };
-
 const loadLocalesMock = jest.fn(
   (locale: string, cb: (error?: Error, data?: Candies) => void) =>
-    cb(undefined, candies[locale])
+    new Promise(resolve => {
+      setTimeout(() => {
+        resolve();
+        cb(undefined, candies[locale]);
+      }, 10);
```

Comment:
?

Reply:
Should I leave it out?

Reply:
I was wondering why it's added? To end up on another tick?

Reply:
Yeah, the intent was just to make it clear that it's an async execution.

Reply:
You can just to `Promise.resolve` then? That's scheules a mircrotask whereas `setTimeout` is a regular task.

Reply:
`Promise.resolve` does not work in this case, because we resolve the loading through a callback.

I can change the loading part though, to use promises only.

Reply:
Done, it's also nicer now! https://github.com/commercetools/merchant-center-application-kit/pull/860/commits/5b36f1f3d527f3266c752d5760b34bfa0258e7bf